crypto/evp/m_sigver.c

```diff
@@ -135,6 +135,14 @@ int EVP_DigestSignFinal(EVP_MD_CTX *ctx, unsigned char *sigret,
     return 1;
 }
 
+int EVP_DigestSign(EVP_MD_CTX *ctx, unsigned char *sigret, size_t *siglen,
+                   const unsigned char *tbs, size_t tbslen)
+{
+    if (sigret != NULL && EVP_DigestSignUpdate(ctx, tbs, tbslen) <= 0)
+        return 0;
+    return EVP_DigestSignFinal(ctx, sigret, siglen);
+}
```

Comment:
The documentation indicates:

> For algorithms which do not support streaming (e.g. PureEdDSA) it is the only way to sign data.

But this looks like a simple wrapper, so I'm wondering if that documentation statement is true?